cmd/gateway/main.go

```diff
@@ -81,17 +81,17 @@ func init() {
 func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	setupDir, err := filepath.Abs(confDir)
 	if err != nil {
-		return err
+		return Error.Wrap(err)
```

Comment:
This cleanup is awesome.  It would be even more awesome if it were in another PR, to make live easier on reviewers.  :D

Reply:
I try to avoid to make more changes than needed, but if the cost of opening a ticket is higher than doing the change like this one, I go ahead.

I usually create new branches for those and separated PRs, however I don't like to spend a lot of time resolving conflicts and if those changes are probably going to conflict with the ones which I was working on, I don't do; in other code sources, being possible to rebase help me a lot on having separated branches and the one which could have conflicts be on top of those, but here rebase aren't allow once the branch is pushed, because it requires to force push, so I don't do.

To wrap, I agree, but there are limitations and only do if it doesn't complicated the work.

NOTE I cannot say if that applies to those changes in this branch, but that's what stop me to act how I did in other code bases.